dash-pipeline/SAI/templates/utils.cpp.j2

```diff
@@ -217,29 +217,155 @@ sai_status_t sai_create_switch_dummy(
     return SAI_STATUS_SUCCESS;
 }
 
-sai_status_t sai_get_switch_attribute_dummy(
+#define DASH_BMV2_NUM_PORTS 2 
+#define DASH_BMV2_CPU_QOS_NUMBER_OF_QUEUES 0
+
+#define OJBTYPE_SHFT 48
```

Comment:
is this OBJTYPE_SHIFT?

Reply:
Used to construct an OID from object type and instance value. See macro `MAKE_OID`